improver/constants.py

```diff
@@ -47,49 +47,49 @@
 HOURS_IN_DAY = 24
 DAYS_IN_YEAR = 365
 
-# 0 Kelvin in degrees C
+#: 0 Kelvin in degrees C
 ABSOLUTE_ZERO = -273.15
 U_ABSOLUTE_ZERO = "celsius"
 
-# Specific gas constant for dry air (J K-1 kg-1)
+#: Specific gas constant for dry air (:math:`J\, K^{-1}\, kg^{-1}`)
 R_DRY_AIR = 287.0
 U_R_DRY_AIR = "J K-1 kg-1"
 
-# Specific gas constant for dry air per mole (J K-1 mol-1)
+#: Specific gas constant for dry air per mole (:math:`J\, K^{-1}\, mol^{-1}`)
 R_DRY_AIR_MOL = 8.314
 U_R_DRY_AIR_MOL = "J K-1 mol-1"
 
-# Specific gas constant for water vapour (J K-1 kg-1)
+#: Specific gas constant for water vapour (:math:`J\, K^{-1}\, kg^{-1}`)
 R_WATER_VAPOUR = 461.6
 U_R_WATER_VAPOUR = "J K-1 kg-1"
 
-# Specific heat capacity of dry air (J K-1 kg-1)
+#: Specific heat capacity of dry air (:math:`J\, K^{-1}\, kg^{-1}`)
 CP_DRY_AIR = 1005.0
 U_CP_DRY_AIR = "J K-1 kg-1"
 
-# Specific heat capacity of water vapour (J K-1 kg-1)
+#: Specific heat capacity of water vapour (:math:`J\, K^{-1}\, kg^{-1}`)
 CP_WATER_VAPOUR = 1850.0
 U_CP_WATER_VAPOUR = "J K-1 kg-1"
 
-# Triple Point of Water (K)
+#: Triple Point of Water (:math:`K`)
 TRIPLE_PT_WATER = 273.16
 U_TRIPLE_PT_WATER = "K"
 
-# Latent heat of condensation of water at 0C (J kg-1)
+#: Latent heat of condensation of water at 0C (:math:`J\, kg^{-1}`)
 LH_CONDENSATION_WATER = 2.501e6
 U_LH_CONDENSATION_WATER = "J kg-1"
-# Molar mass of water vapour (kg mol-1)
+#: Molar mass of water vapour (:math:`kg\, mol^{-1}`)
 WATER_VAPOUR_MOLAR_MASS = 0.01801
 U_WATER_VAPOUR_MOLAR_MASS = "kg mol-1"
-# Latent heat temperature dependence (J K-1 kg-1); from Met Office UM.
-# Applied to temperatures in Celsius: LH = 2501 - 2.34E3 * T(celsius)
+#: Latent heat temperature dependence (:math:`J\, K^{-1}\, kg^{-1}`); from Met Office UM.
+#: Applied to temperatures in Celsius: :math:`LH = 2501 - 2.34\times10^3 * T(celsius)`
 LATENT_HEAT_T_DEPENDENCE = 2.34e3
 U_LATENT_HEAT_T_DEPENDENCE = "J K-1 kg-1"
 
-# Repsilon, ratio of molecular weights of water and dry air (Earth)
+#: Repsilon, ratio of molecular weights of water and dry air (Earth)
 EARTH_REPSILON = 0.62198
 U_EARTH_REPSILON = "1"
 
-# Dry Adiabatic Lapse Rate (DALR) in unit of K m-1
+#: Dry Adiabatic Lapse Rate (DALR) in unit of :math:`K\, m^{-1}`
```

Comment:
Thanks for adding the constants to the documentation, that is a useful addition. However, I think we should drop the latex rendering of the units. It does not have a major impact on the final rendering (some super-scripting and italicisation) but makes the code view far less readable. I include below examples of the web-rendered view of this file with and without the latex change to demonstrate the relatively minimal benefit.

With this change:
![image](https://user-images.githubusercontent.com/28042362/177129420-1e195f88-5510-4637-a039-cc429d84a2b8.png)

Without this change:
![image](https://user-images.githubusercontent.com/28042362/177129357-bc068c22-0a28-4d71-a124-9b3d823bf46b.png)

So what do you think about keeping the `#:` which are making the file render but dropping the latex?

Reply:
I have changed all these back to plain text to be easier to read in the Python module. I have also removed the unused `U_{constant_name}` strings, as they were unused, with one exception which is now hard-coded.